Make all release notes past tense (#942)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

* The `subctl cloud prepare` command no longer causes errors if the list of ports is empty.
* `subctl` operations which deploy images now allow those images to be overridden. The overrides are specified using `--image-override`:
  * `subctl benchmark`.
  * `subctl verify`.
  * `subctl diagnose` sub-commands.
* The namespace is now included in `EndpointSlice` names to avoid conflicts between services with the same name in multiple namespaces.
* Bumped go-restful to 2.16.0 to address CVE-2022-1996.
* Bumped k8s.io/client-go to 0.20.15 to fix CVE-2020-8565.
* Bumped golang.org/x/crypto to 0.6.0 to fix CVE-2022-27191.
* Bumped golang.org/x/net to 0.7.0 to fix a number of security issues.

## v0.13.3

This is a bugfix release:

* The `subctl diagnose kube-proxy-mode` command now works with different versions of iproute packages.
* The following changes were made to pods running `subctl diagnose` commands in order to allow them to run commands like `tcpdump`:
  * Made the `diagnose` pod privileged.
  * Run the `diagnose` pod with user ID 0.

## v0.12.3

This is a bugfix release:

* Image version hashes are now 12 character long, avoiding possible collisions between images.

> * Stoped using cluster-owned tag for AWS cloud prepare, fixing problems with Submariner security groups left over after uninstallation.

Reviewer: ```suggestion
* Stopped using cluster-owned tag for AWS cloud prepare, fixing problems with Submariner security groups left over after uninstallation.
```

This is a bugfix release:

> * Stoped using cluster-owned tag for AWS Security Group lookup.

Reviewer: ```suggestion
* Stopped using cluster-owned tag for AWS Security Group lookup.
```

* Running the `subctl diagnose firewall` command with individual kubeconfigs will now deploy diagnose pods in the `submariner-operator` namespace
to avoid pod security errors.
* The periodic public IP watcher is enhanced to use random external servers to resolve the public IP associated with Gateway nodes.
* The `subctl diagnose kube-proxy-mode` command now works with different versions of iproute packages.
* The following changes were made to pods running `subctl diagnose` commands in order to allow them to run commands like `tcpdump`:
  * Made the `diagnose` pod privileged.
  * Run the `diagnose` pod with user ID 0.

## v0.13.2

* Added support for OpenShift 4.12.
* Service Discovery now returns a DNS error message in the response body when no matching records are found when queried about
  `clusterset.local`. This prevents unnecessary retries.

> * Stoped using cluster-owned tag for AWS Security Group lookup.
> * Stoped using api.ipify.org as the first resolver for public IPs.

Reviewer: ```suggestion
* Stopped using cluster-owned tag for AWS Security Group lookup.
* Stopped using api.ipify.org as the first resolver for public IPs.
```

## v0.8.1

* Submariner Gateway Health Check is now supported with Globalnet deployments.

> * Supported deploying OVN in kind using `make clusters using=ovn` for E2E testing and development environments.
> * Supported debugging the Libreswan cable driver.

Reviewer: ```suggestion
* Added support for deploying OVN in kind using `make clusters using=ovn` for E2E testing and development environments.
* Added support for debugging the Libreswan cable driver.
```

* Fixed the cable driver label in the Prometheus latency metrics.

> * Supported non-TLS connections for OVN databases.

Reviewer: ```suggestion
* Added support for non-TLS connections for OVN databases.
```

* Support for `verify-connectivity` checks between two connected clusters.
* Deployment of Submariner gateways based on `DaemonSet` instead of `Deployment`.
* Renamed `submariner` Pods to `submariner-gateway` Pods for clarity.
* Print version details on crash (`subctl`).

> * Stoped storing IPsec key on Broker during `deploy-broker`, now it's only contained into the `.subm` file.

Reviewer: ```suggestion
* Stopped storing IPsec key on Broker during `deploy-broker`, now it's only contained into the `.subm` file.
```
